geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStore.scala

```diff
@@ -146,64 +143,66 @@ class AccumuloDataStore(val connector: Connector, override val config: AccumuloD
     import SimpleFeatureTypes.Configs.{ENABLED_INDEX_OPTS, ENABLED_INDICES}
     import SimpleFeatureTypes.InternalConfigs.{INDEX_VERSIONS, SCHEMA_VERSION_KEY}
 
-    var sft = super.getSchema(typeName)
+    val escapedTypeName = typeName.replace("~", ESCAPE)
```

Comment:
could we move the handling of this into the metadata class? I think that would be a more logical place, since the metadata implementation is what's causing the `~` to be invalid